dht-node/src/dht_node/index.test.ts

```diff
@@ -44,10 +44,11 @@ const lookupResultMock = {
   token: Buffer.from(TEST_TOPIC),
   from: {
     id: Buffer.from('somone'),
+    foreign: true,
```

Comment:
is this part of the DHT library? Because this is DHT-related?
Isn't `foreign` our new database key?